Exercise3/Models/Ultilities.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Exercise3.Models
+{
+    public class Ultilities
+    {
+        public static List<ManagerItem> GetFileAndFolderNames(string path)
+        {
+            var mList = new List<ManagerItem>();
+            var fileAndFolderNames = Directory.GetFileSystemEntries(path);
```

Comment:
On Linux system (which android actually extended), everything is files, including files, folders, drives. So the name here is `fileNames`

Reply:
done